frontend/src/App.tsx

```diff
@@ -1,11 +1,17 @@
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import reactLogo from './assets/react.svg'
 import './App.css'
 import Connection from './components/connection'
 
 function App() {
```

Comment:
pls put this file in another component/service/something.

Reply:
(as in stuff inside of this, as this should be for the home page component or w/e)